extensions/amp-script/0.1/amp-script.js

```diff
@@ -117,19 +118,14 @@ export class AmpScript extends AMP.BaseElement {
      *   2. The amp-script tag or any of its parents must also have 'data-ampdevmode'.
      */
     const devTag = 'data-ampdevmode';
-    let parentHasDevTag = false;
-    if (this.element.ownerDocument.documentElement.hasAttribute(devTag)) {
-      let parentWalker = this.element;
-      while (parentWalker != null) {
-        if (parentWalker.hasAttribute('data-ampdevmode')) {
-          parentHasDevTag = true;
-          break;
-        }
-        parentWalker = parentWalker.parentNode;
-      }
-    }
+    const doc = this.element.ownerDocument.documentElement;
```

Comment:
Nit: `doc` implies the `#document` node, not the `<html>` element.

Reply:
renaming to `htmlEl`